spec/features/pictures_feature_spec.rb

```diff
@@ -0,0 +1,164 @@
+require 'rails_helper'
+
+feature 'pictures' do
+  context 'no pictures have been added' do
+    scenario 'should display a prompt to add a restaurant' do
+      visit '/pictures'
+      expect(page).to have_link 'Add a picture'
+      expect(page).to have_content 'No pictures yet'
+    end
+  end
+
+  context 'pictures have been added' do
+    before do
+      Picture.create(name:'test',
+                     avatar_file_name: 'test.jpg',
+                     avatar_file_size: '10',
+                     avatar_content_type: 'image/jpeg')
+    end
+
+    scenario 'display pictures' do
+      visit '/pictures'
+      expect(page).to have_content('test')
+      expect(page).not_to have_content('No pictures yet')
+    end
+  end
+
+  context 'viewing pictures' do
+
+    let!(:test){Picture.create(name:'test',
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Space missing after colon.